src/lib/logger.js

```diff
@@ -158,14 +90,14 @@ class Logger {
 
     this._notifiers
       .forEach(notifier => {
-        if (!notifier.canOutput(log) || !notifier.options.enabled) {
-          return;
-        }
+        if (!notifier.canOutput(log)) return;
+        if (!notifier.options.enabled) return;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
        if (!notifier.options.enabled) {return;}
```